stock_prelocation_putaway/README.rst

```diff
@@ -0,0 +1,114 @@
+==========================
+Stock Pre-location Putaway
+==========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fwms-lightgray.png?logo=github
+    :target: https://github.com/OCA/wms/tree/14.0/stock_prelocation_putaway
+    :alt: OCA/wms
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-stock_prelocation_putaway
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/285/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to use pre-sort locations.
+
+For instance, in a wherehouse with alleys named A,B,C and D:
```

Comment:
`wherehouse` c pas rillieux de janeiro chez toi ?

Reply:
Fixed thanks ;)

Reply:
Still not fixed it seems, not pushed? :)

Reply:
It's fixed in DESCRIPTION.rst line:3. The readme has not bee re-generated.